Week3/homework/addToShoppingCart.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+function addToShoppingCart(item) {
+    // An array with two Items
+    var array = ['bananas', 'milk'];
```

Comment:
1. Use `let`
2. Use a more descriptive name than `array`, perhaps `shoppingCart` works better.

Reply:
Thank you for the feed back I fixed